src/vm/excep.cpp

```diff
@@ -7164,6 +7164,13 @@ EXTERN_C void JIT_MemCpy_End();
 
 EXTERN_C void JIT_WriteBarrier_End();
 EXTERN_C void JIT_CheckedWriteBarrier_End();
+#else
+#ifdef FEATURE_PAL
+EXTERN_C void JIT_WriteBarrierGroup();
```

Comment:
Ah, so you did that just so that you can use the CHECK_RANGE macro. I was thinking that for x86, you would just put in a direct comparison like we have in AdjustContextForWriteBarrier for x86.
It is a bit confusing to have two different sets of symbols for the same address range check, but I like your naming better.
Could you please remove the JIT_WriteBarrierStart and JIT_WriteBarrierLast / JIT_PatchedWriteBarrierStart and JIT_PatchedWriteBarrierLast and replace their usage by your new symbols then? This would include renaming them in jithelp.asm too. There are just few places to make this change.